modules/virt-reclaiming-persistent-volumes-for-binding-to-a-new-persistent-volume.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// virt/virtual_machines/virtual_disks/virt-reclaiming-persistent-volumes.adoc
+
+[id="virt-reclaiming-persistent-volumes-for-binding-to-a-new-persistent-volume_{context}"]
+
+= Reclaiming persistent volumes for binding to a new persistent volume
+
+Use this procedure to reclaim a statically-provisioned persistent volume for reuse.
+
+.Procedure
+. Enter the following command to display connection details for the persistent volume
+you want to reclaim. Save the output for future reference.
++
+----
+$ oc describe pv <pv_name>
+----
+. Review the `storageClass` object. Verify that the Storage Class `reclaimPolicy` is set to `delete`.
```

Comment:
Does creating a new storageClass impact existing PVs?

Reply:
Might be something to check later. I can open a bug. This is what Steve recommended we do with all additional content changes today